src/stream/src/executor/backfill/no_shuffle_backfill.rs

```diff
@@ -506,10 +506,11 @@ where
             )
             .await?;
```

Comment:
nits: we can use `map` to call `KeyedRow::into_owned_row` to avoid introdcuing the `owned_row_iter` method.